bundle/manifests/kogito-operator.clusterserviceversion.yaml

```diff
@@ -59,8 +59,8 @@ metadata:
     containerImage: quay.io/kiegroup/kogito-operator:2.0.0-snapshot
     createdAt: "2019-08-22T13:12:22Z"
     description: Kogito Operator for deployment and management of Kogito services.
-    operators.operatorframework.io/builder: operator-sdk-v1.2.0
-    operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
+    operators.operatorframework.io/builder: operator-sdk-v1.10.0+git
```

Comment:
1.11.0?

Reply:
`operator-sdk-v1.10.0+git` label is for `1.11.0`.